test/test_assignment.py

```diff
@@ -83,17 +83,42 @@ def test_setdelayed_oneidentity():
             "{F[a, b], H[a, b], F[a, b]}",
             None,
         ),
+        (
```

Comment:
I added this "idiom" to tell to the test routine that a refresh is needed between tests. Maybe there is a more canonical way to to this, but for now it works. @rocky, @TiagoCavalcante, do you have an idea about how to do this better?